app/services/recaptcha_validator.rb

```diff
@@ -33,12 +40,14 @@ def exempt?
     !score_threshold.positive?
   end
 
-  def valid?(recaptcha_token)
-    return true if exempt?
-    return false if recaptcha_token.blank?
+  # @return [Array(Boolean, String), Array(Boolean, nil)]
+  def validate(recaptcha_token)
```

Comment:
I'm wondering if it might be worth refactoring this as one of our "form" classes, and then this can be `submit(recaptcha_token:)` and return `assessment_id` as part of `FormResponse` `extra`?

Reply:
Refactoring completed separately in #10540 and #10549, and rebased this branch to use the new form classes.